docs/devguide/create-metricset.asciidoc

```diff
@@ -239,15 +269,24 @@ var (
 			"test_obj_string": c.Str("testObjString"),
 		},
 	}
+
+	additionalSchema = s.Schema{
+		"second_string": c.Str("secondString"),
+		"second_int": ": c.Int("secondInt"),
+	}
 )
 
-func eventMapping(input map[string]interface{}) common.MapStr {
-	return schema.Apply(input)
-}
+	data, err := schema.apply(input)
+	if err != nil {
+		return err
+	}
+
+	if m.parseMoreData{
+		additionalSchema.ApplyTo(data, input)
```

Comment:
Error handling for this example? `ApplyTo` can be used to apply multiple schemas, but also to do more fine-grained error handling, as it returns the raw multierror.